Wording for FAQ and landing page

mcdurdin opened the pull request:
Now ready for review.

Previously: The FAQ page is not yet routed; can I leave that for you, @SeriousMatt? There are a bunch of incomplete references etc in that file which I am working on clarifying, but I wanted to get the basic structure there.

mcdurdin committed:
Draft wording for FAQ and landing page

PaulNel reviewed (approved):
Changes look good to me. Thanks for putting these together!

SeriousMatt committed:
Merge branch 'master' into wording

SeriousMatt committed:
Fix merge

mcdurdin commented:
Note: this is not ready for merge yet; I still need to fixup links.

SeriousMatt committed:
Rearrange application folders for routing for faq page

SeriousMatt reviewed (approved):
@mcdurdin The routing for the faq page (/faq) is now set up.

mcdurdin committed:
Merge branch 'master' into wording

mcdurdin committed:
Fixup zero-decimal currencies and refactor currency data into separate unit; cleanup app.component.ts bad merge

mcdurdin committed:
Start formatting

mcdurdin committed:
Formatting and content

mcdurdin commented:
@SeriousMatt this is now ready for review; added header, footer, ways-to-give, tidied up formatting of FAQ, fixed links and content, added very basic navigation. I think it's pretty much good enough to go now.

SeriousMatt committed:
Fix spacing and other lint issues

SeriousMatt reviewed (approved):
@mcdurdin works locally for me and passing specs. I'll merge this in and get it deployed.

SeriousMatt committed:
Ignore dist folder

SeriousMatt committed:
Fix merge issues

SeriousMatt commented on src/app/app.component.ts:
```diff
@@ -9,219 +9,6 @@ import { Component, OnInit } from '@angular/core';
-          success => {
-            // Handle result
-            console.log('POST call successful value returned in body', success);
-            self.successMessage = 'The transaction has been completed. Thank you for you donation!';
```
@mcdurdin here is some the missing code from the merge. This occurred because we changed file names.
